sonarts-core/scripts/NewRule.sln

```diff
@@ -0,0 +1,25 @@
+﻿

```

Comment:
This file (and nsproj) are useful to keep so the script can be built/debugged in Visual Studio